src/nls/root/strings.js

```diff
@@ -281,6 +281,9 @@ define({
     "STATUSBAR_CODE_INSPECTION_TOOLTIP"     : "{0}. Click to toggle report panel.",
     "STATUSBAR_DEFAULT_LANG"                : "(default)",
     "STATUSBAR_SET_DEFAULT_LANG"            : "Set as Default for .{0} Files",
+    "STATUSBAR_LINE_ENDINGS_CRLF"           : "CRLF",
+    "STATUSBAR_LINE_ENDINGS_LF"             : "LF",
+    "STATUSBAR_LINE_ENDINGS_TOOLTIP"        : "Click to toggle between CRLF(Windows) and LF(Linux and Mac) line endings",
```

Comment:
There should be a space before each "(".  But we may also want to consider simplifying this -- something shorter like "Click to toggle line endings" may be fine.